pyccel/epyccel.py

```diff
@@ -215,40 +276,21 @@ def epyccel( python_function_or_module, **kwargs ):
     """
     Accelerate Python function or module using Pyccel in "embedded" mode.
 
+    This function accelerates a Python function or module using Pyccel in "embedded" mode.
+    It generates optimized code in the specified language (default is 'fortran')
+    and compiles it for improved performance
+
     Parameters
     ----------
     python_function_or_module : function | module
         Python function or module to be accelerated.
-
-    verbose : bool
-        Print additional information (default: False).
-
-    language : {'fortran', 'c', 'python'}
-        Language of generated code (default: 'fortran').
-
-    accelerators : iterable of str, optional
-        Parallel multi-threading acceleration strategy
-        (currently supported: 'mpi', 'openmp', 'openacc').
-
-    Options for parallel mode
-    -------------------------
-    comm : mpi4py.MPI.Comm, optional
-        MPI communicator for calling Pyccel in parallel mode (default: None).
-
-    root : int, optional
-        MPI rank of process in charge of accelerating code (default: 0).
-
-    bcast : {True, False}
-        If False, only root process loads accelerated function/module (default: True).
-
-    Other options
-    -------------
-    compiler : str, optional
-        User-defined command for compiling generated source code.
+    **kwargs :
+        Additional keyword arguments for configuring the compilation and acceleration process.
+        Available options are defined in epyccel_seq.
```

Comment:
Can we add `epyccel_seq` to `See Also`. That way when we auto-generate docs from these docstrings (see #1287) there will be a link to the place where the kwargs are defined

Reply:
Typo `extenstion`->`extension` but I'm not sure this definition is accurate. I suggest:
```
     See Also 
     -------- 
     epyccel_seq
          The version of this function called in a sequential context.
```

Reply:
@mkaddani Don't miss this slightly hidden comment

Reply:
fixed: ab257f9c